scripts/gen_eval_files.lua

```diff
@@ -264,15 +264,15 @@ local function get_api_keysets()
   local keysets = metadata.keysets
 
   for _, keyset in ipairs(keysets) do
-    local kname = keyset[1]
-    local kdef = keyset[2]
-    for _, field in ipairs(kdef) do
-      field[2] = api_type(field[2])
+    local name, keys, types = unpack(keyset)
```

Comment:
LuaLs can't analyse this properly so I usually write this as:

```suggestion
    local name, keys, types = keyset[1], keyset[2], keyset[3]
```

Reply:
shouldn't luals be fixed then? This is a fundamental pattern of the language.

Reply:
Yes, so we don't do that.  Not using unpack negates the point of using tuple packing in the first place. Where more structure is needed I would switch to named keys instead.

Reply:
This is clearly a hill you're going to die on.

Go ahead and disregard my comment.